### PR TITLE
:robot: :memo: Restructure Project

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Restructure the file structure for the course content

### Why
Cleaner this way, but will also make it easier to "scale" (for lack of a better term). 

### Where
Everywhere 

### How
Pain

### Testing
Yes.

### CODE REVIEW

File: site/index.rst [hunk 1]
```diff
@@ -142,75 +142,78 @@ Getting Started
...
-    topic12
-    topic13
-    topic13b
-    topic14
-    topic15
-    topic16  
-    topic17
-    topic18
-    topic19
+    topics/intro/intro
```
Comment (twentylemon) [inline site/index.rst:154]: Minor, does rst support "index" files? Like when you navigate to `whatever.com`, you actually go to `whatever.com/index.html`. Would be a nice cleanup here to only say `topics/intro`.

Comment (jameshughes89) [reply site/index.rst:154]: I had tried this. Sadly, no dice.

Comment (jameshughes89) [reply site/index.rst:154]: actually --- hold on, I have one last idea.

Comment (jameshughes89) [reply site/index.rst:154]: the new idea failed too

**Pull request merged**